USAGE_android.md

```diff
@@ -413,8 +898,67 @@ Key(s) | Key code(s) | Action
 Space, p | Space = 62, p = 44 |Toggle pause/play.
 D-pad right, n | D-pad right = 22, n = 42 | Advance to the next frame when paused.
 
-### Virtual Swapchain
 
-During replay, swapchain indices for present can be different from captured indices. Causes for this can include the swapchain image count differing between capture and replay, and `vkAcquireNextImageKHR` returning a different `pImageIndex` at replay to the one that was captured. These issues can cause unexpected rendering or even crashes.
+### Limitations of Replay On Android
+
+The current version of the Android replay layer has the following limitations:
+
+  * Creation of more than one swapchain at a time is not guaranteed to work.
+    The replay tool has a single ANativeActivity with a single ANativeWindow.
+    Android may enforce a limitation of one Vulkan surface per-window.
+    It may be possible to support this case with future improvements to the
+    replay tool by extending the use of Java-based components to create multiple
+    activities or surfaces.
+  * Permission to access external storage must be granted manually.
+    Future improvements to the replay tool may add support for prompting the
+    user to grant permission at run time when the tool detects that the
+    permission has not been previously granted.
+  * The replay tool should be ‘force-stopped’ between consecutive runs.
+    Improvements may be required to the replay tool’s handling of
+    start/resume/pause/stop events to ensure that all startup and shutdown
+    events are processed correctly.
+
+
+### Troubleshooting Replay of Applications
+
+There are several steps to take to determine why a replay of a capture may not
+be working.
+Attempt each of these and re-run after the suggested modifications to see if
+the problem resolves.
+
+#### Removing Unsupported Extensions
+
+When an application is performing a capture, there is a main difference from
```

Comment:
"When capturing an application, the GFXReconstruct layer exposes the `VK_EXT_tooling_info` extension."